yw_website/yw_website/apps/website/static/design/edit_page_design.css

```diff
@@ -1,7 +1,15 @@
 .edit {
 	margin-left: 3%;
 	margin-right: 3%;
+	padding: 3%;
+	box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2); /* this adds the "card" effect */
 	text-align: left;
+	background-color: white;
+}
+
+h3 {
```

Comment:
sorry I missed this, but we should not be applying styling to html elements if we dont have too, this should go in a class and the h3 that needs this styling would use this class